### PR TITLE
Add Deploy to Azure links to intro tutorials

### LINKED ISSUE #25138: Add links to  Tutorial: Deploy an ASP.NET Core and Azure SQL Database app to Azure App Service
Add links to https://docs.microsoft.com/en-us/azure/app-service/tutorial-dotnetcore-sqldb-app to the end of all on boarding tutorials.

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #25138 

Internal Review links:
[First-web-api](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/first-web-api?view=aspnetcore-6.0&branch=pr-en-us-25665&tabs=visual-studio)
[min-web-api](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/min-web-api?view=aspnetcore-6.0&branch=pr-en-us-25665&tabs=visual-studio)
[SignalR](https://review.docs.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-6.0&branch=pr-en-us-25665&tabs=visual-studio)
[Build-a-blazor](https://review.docs.microsoft.com/en-us/aspnet/core/blazor/tutorials/build-a-blazor-app?view=aspnetcore-6.0&branch=pr-en-us-25665&pivots=server)

Verified Razor Pages and MVC intro tutorials already had the links

Added links to either one of these:

- ASP.NET plus SQL scenarios:  [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](https://docs.microsoft.com/en-us/azure/app-service/tutorial-dotnetcore-sqldb-app) in the introduction tutorials.
- ASP.NET without a DB: [Quickstart: Deploy an ASP.NET web app](https://docs.microsoft.com/en-us/azure/app-service/quickstart-dotnetcore)

Followed the existing example in MVC tutorial of adding a new _Publish to Azure_ section at the end of the topic for version 6.x and 5.x

### CODE REVIEW

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md [hunk 1]
```diff
@@ -174,6 +174,10 @@ cd TodoList
 
 1. When finished, shut down the app in the command shell. Many command shells accept the keyboard command <kbd>Ctrl</kbd>+<kbd>C</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>C</kbd> (macOS) to stop an app.
 
+### Publish to Azure
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:177]: @guardrex

**Review by Rick-Anderson — COMMENTED** [review]
None of these use SQL, which complicates deployment. Don't we have a link that deploys the web app without SQL?

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md (hunk 1 above)
Comment (guardrex) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:177]: Can we go H2 here?

```suggestion
## Publish to Azure
```

**Review by guardrex — APPROVED** [review]

**Comment by wadepickett** [comment]
@Rick-Anderson,  there is a better topic to link to.  I will redo and point to this instead: /azure/app-service/quickstart-dotnetcore

Since you wanted a that specific link for those topics, I thought it was already determined that was the best we had, but I should have checked around anyway.  I didn't realize we had one that did not include a db.  Thanks.

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md [hunk 2]
```diff
@@ -174,6 +174,10 @@ cd TodoList
 
 1. When finished, shut down the app in the command shell. Many command shells accept the keyboard command <kbd>Ctrl</kbd>+<kbd>C</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>C</kbd> (macOS) to stop an app.
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/tutorial-dotnetcore-sqldb-app).
```
Comment (wadepickett) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:179]:
```suggestion
For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md [hunk 3]
```diff
@@ -351,6 +355,10 @@ cd TodoList
...
```
Comment (wadepickett) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:360]:
```suggestion
For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/first-web-api.md [hunk 4]
```diff
@@ -1625,6 +1625,10 @@ See [Tutorial: Call an ASP.NET Core web API with JavaScript](xref:tutorials/web-
 
 [!INCLUDE[](~/includes/DuendeIdentityServer.md)]
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/tutorial-dotnetcore-sqldb-app).
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-web-api.md:1630]:
```suggestion
For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/signalr.md [hunk 5]
```diff
@@ -258,6 +258,11 @@ The preceding highlighted code adds SignalR to the ASP.NET Core dependency injec
 >   dotnet dev-certs https --clean
 >   dotnet dev-certs https --trust
 >   ```
+
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/tutorial-dotnetcore-sqldb-app).
```
Comment (wadepickett) [inline aspnetcore/tutorials/signalr.md:264]:
```suggestion
For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/signalr.md [hunk 6]
```diff
@@ -480,4 +485,8 @@ The SignalR server must be configured to pass SignalR requests to SignalR.
 >   dotnet dev-certs https --trust
 >   ```
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/tutorial-dotnetcore-sqldb-app).
```
Comment (wadepickett) [inline aspnetcore/tutorials/signalr.md:490]:
```suggestion
For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md [hunk 7]
```diff
@@ -174,6 +174,10 @@ cd TodoList
 
 1. When finished, shut down the app in the command shell. Many command shells accept the keyboard command <kbd>Ctrl</kbd>+<kbd>C</kbd> (Windows) or <kbd>⌘</kbd>+<kbd>C</kbd> (macOS) to stop an app.
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```
Comment (Rick-Anderson) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:179]: Doesn't title need to change? No SQL now

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md (hunk 7 above)
Comment (wadepickett) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:179]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md [hunk 8]
```diff
@@ -351,6 +355,10 @@ cd TodoList
...
```
Comment (wadepickett) [inline aspnetcore/blazor/tutorials/build-a-blazor-app.md:360]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/first-web-api.md [hunk 9]
```diff
@@ -1625,6 +1625,10 @@ See [Tutorial: Call an ASP.NET Core web API with JavaScript](xref:tutorials/web-
 
 [!INCLUDE[](~/includes/DuendeIdentityServer.md)]
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```
Comment (wadepickett) [inline aspnetcore/tutorials/first-web-api.md:1630]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/min-web-api.md [hunk 10]
```diff
@@ -460,6 +460,10 @@ The preceding code uses [web defaults](/dotnet/standard/serialization/system-tex
 
 For an example of testing a minimal API app, see [this GitHub sample](https://github.com/davidfowl/CommunityStandUpMinimalAPI/blob/main/TodoApi.Tests/TodoTests.cs).
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```
Comment (wadepickett) [inline aspnetcore/tutorials/min-web-api.md:465]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/signalr.md [hunk 11]
```diff
@@ -258,6 +258,11 @@ The preceding highlighted code adds SignalR to the ASP.NET Core dependency injec
 >   dotnet dev-certs https --clean
 >   dotnet dev-certs https --trust
 >   ```
+
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```
Comment (wadepickett) [inline aspnetcore/tutorials/signalr.md:264]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

File: aspnetcore/tutorials/signalr.md [hunk 12]
```diff
@@ -480,4 +485,8 @@ The SignalR server must be configured to pass SignalR requests to SignalR.
 >   dotnet dev-certs https --trust
 >   ```
 
+## Publish to Azure
+
+For information on deploying to Azure, see [Tutorial: Build an ASP.NET Core app in Azure with SQL Database](/azure/app-service/quickstart-dotnetcore).
```
Comment (wadepickett) [inline aspnetcore/tutorials/signalr.md:490]:
```suggestion
For information on deploying to Azure, see [Quickstart: Deploy an ASP.NET web app](/azure/app-service/quickstart-dotnetcore).
```

Comment (Rick-Anderson) [reply aspnetcore/tutorials/signalr.md:490]: @wadepickett  why all the commits and suggestions? Can't you fix everything in one commit?

File: aspnetcore/blazor/tutorials/build-a-blazor-app.md (hunk 7 above)
Comment (wadepickett) [reply aspnetcore/blazor/tutorials/build-a-blazor-app.md:179]: Ugh.  Yes, of course.  ouch.  Thanks.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/tutorials/signalr.md (hunk 12 above)
Comment (wadepickett) [reply aspnetcore/tutorials/signalr.md]: @Rick-Anderson I can.  If this is a big thing for you I can change the way I work.  I usually work from a local clone.  Today I had a family member going to the hospitaL.  I ended up having to quickly work out of gitHub which I don't usually do, and so today it was different.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/signalr.md]: >  I ended up having to quickly work out of gitHub which I don't usually do, and so today it was different.

It's much faster and more efficient to edit it the browser and do a global replace, then commit. Less work, less noise.

Comment (Rick-Anderson) [reply aspnetcore/tutorials/signalr.md]: image

**Pull request merged**